rfcs/0000-plan-dynanism.md

```diff
@@ -0,0 +1,261 @@
+---
+feature: plan-dynamism
+start-date: 2019-02-01
+author: John Ericson (@Ericson2314)
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
```

Comment:
I think this section could use a paragraph actually describing the benefit. You list some problems that are hard to solve in the Motivation but a succinct summary would be useful. Possibly something like:

*Plan Dynamism would allow making Nix derivations available for large numbers of packages from language-specific package repositories such as NPM or crates.io with very little per-package effort*